device-types/Zyxel/PMG1005-T20B.yaml

```diff
@@ -0,0 +1,23 @@
+---
+manufacturer: Zyxel
+model: PMG1005-T20B
+slug: zyxel-pmg1005-t20b
+part_number: 'SKU : PMG1005-T20C'
```

Comment:
It looks like the T20B and T20C are different models, is that true? If so they should be different device types

Also, please make sure the part number does not have `SKU : ` in it. It should just be the applicable part number

Reply:
done